include/geos/algorithm/distance/PointPairDistance.h

```diff
@@ -108,6 +108,10 @@ class PointPairDistance
 			initialize(p0, p1, dist);
 	}
 
+        bool getIsNull() {
+                return isNull;
+        }
+
```

Comment:
The indent style of this block seems not to match the previous block, please check